opennms-doc/guide-install/src/asciidoc/text/opennms/debian.adoc

```diff
@@ -0,0 +1,152 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[gi-install-opennms-debian]]
+=== Install on Debian-based systems
+
+This section describes how to install the _OpenNMS_ platform on _Ubuntu 14.04 LTS_.
+The setup process is described in the following steps:
+
+. Install OpenNMS apt repository server with GPG key to verify packages
+. Installation of the _opennms_ meta package which handles all dependencies
+. Initialize _PostgreSQL_ database and configure access
+. Initialize _OpenNMS_ and first start of the application
+
+[[gi-install-opennms-deb-repo]]
+==== Setup OpenNMS Debian repository
+
+.Content of the new _OpenNMS Debian_ repository file `/etc/apt/sources.list.d/opennms.list`
```

Comment:
Better: "Create a new file `/etc/apt/sources.list.d/opennms.list`for the _OpenNMS Debian_ repository:"